tests/cli-misc.rs

```diff
@@ -568,6 +568,40 @@ fn rename_rls_remove() {
     });
 }
 
+#[test]
+fn install_with_profile() {
+    clitools::setup(Scenario::Full, &|config| {
+        set_current_dist_date(config, "2015-01-01");
+        expect_ok(config, &["rustup", "default", "stable"]);
```

Comment:
You don't need this really, it's just wasting time in the test installing a toolchain you do nothing with.